include/slang/binding/AssignmentExpressions.h

```diff
@@ -65,15 +65,19 @@ class AssignmentExpression : public Expression {
 struct CastExpressionSyntax;
 struct SignedCastExpressionSyntax;
 
+enum class ConvertCast : uint8_t { Propagate, Implicit, BitstreamCast, Explicit };
```

Comment:
Could you call this ConversionKind instead?

Reply:
Yes, I will change it.